app/models/deleted_object_entry.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DeletedObjectEntry < ApplicationRecord
```

Comment:
Why "Entry"? It could be just `DeletedObject`, IMO.

Reply:
feel the same.